internal/rbd/rbd_util.go

```diff
@@ -79,6 +79,16 @@ const (
 
 	// krbd attribute file to check supported features.
 	krbdSupportedFeaturesFile = "/sys/bus/rbd/supported_features"
+
+	rbdFeatureLayering      = (1 << 0) // RBD_FEATURE_LAYERING
```

Comment:
these constants/#defines are already available in go-ceph:

- https://github.com/ceph/go-ceph/blob/master/rbd/features.go
- https://github.com/ceph/go-ceph/blob/master/rbd/features_nautilus.go

Better use those.